kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -390,6 +390,7 @@ export default {
             });
         })
         .then(() => {
+          document.dispatchEvent(new CustomEvent(`refresh-kudos-${this.entityId}`));
```

Comment:
Event `exo-kudos-sent` already sent, why you don't use it instead of send new one ?